### PR TITLE
Feature/classifications

### PR DESCRIPTION
**Pull request by AIbrahimQA**

### CODE REVIEW

**Review by SamWhite745 — APPROVED** [review]
All looks good

File: app/controllers/classifications.scala [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package controllers
+
+import javax.inject._
+import play.api.mvc._
+
+@Singleton
+class classifications @Inject()(cc: ControllerComponents) extends AbstractController(cc) {
```
Comment (DannyPennington) [inline app/controllers/classifications.scala]:
```suggestion
class Classifications @Inject()(cc: ControllerComponents) extends AbstractController(cc) {
```
Change casing for consistency

File: conf/routes [hunk 2]
```diff
@@ -2,4 +2,7 @@ GET     /assets/*file               controllers.Assets.versioned(path="/public",
 
 GET     /                           controllers.HomeController.index
 
-GET     /listingGallery             controllers.HomeController.listingGallery
\ No newline at end of file
+GET     /listingGallery             controllers.HomeController.listingGallery
+
+GET     /classifications             controllers.classifications.createClassifications
```
Comment (DannyPennington) [inline conf/routes]:
```suggestion
GET     /classifications             controllers.Classifications.createClassifications
```
Change to match controller name change

**Review by DannyPennington — CHANGES_REQUESTED** [review]
Looks good, just minor change to names to match convention

File: app/controllers/classifications.scala [hunk 3]
```diff
@@ -0,0 +1,13 @@
+package controllers
+
+import javax.inject._
+import play.api.mvc._
+
+@Singleton
+class Classifications @Inject()(cc: ControllerComponents) extends AbstractController(cc) {
```
Comment (SamWhite745) [inline app/controllers/classifications.scala]: Name of the class should be ClassificationsController and rename file to match

**Review by SamWhite745 — CHANGES_REQUESTED** [review]

**Review by SamWhite745 — APPROVED** [review]

**Review by DannyPennington — APPROVED** [review]

**Pull request merged**